capten/cassandra/migrations/003_agent.up.cql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE tekton (
+    id uuid,
+    project_url text,

```

Comment:
instead of project_url, it need to store git_project_id, this value comes when syncronize with git_project_data DB table